q2_fmt/_ancombc.py

```diff
@@ -0,0 +1,86 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2022-2023, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+import pandas as pd
+from qiime2 import Metadata
+
+# TODO: Change Import path.
+from q2_fmt._peds import _check_for_time_column, _check_reference_column
+
+
+def detect_donor_indicators(ctx, table, reference_column, time_column,
+                            baseline_timepoint, metadata,
+                            level_delimiter=None):
+    filter = ctx.get_action('feature_table', 'filter_samples')
+    ancombc = ctx.get_action('composition', 'ancombc')
+    da_barplot = ctx.get_action('composition', 'da_barplot')
+    results = []
+
+    _, _ = _check_for_time_column(metadata.to_dataframe(),
+                                  time_column)
+    _ = _check_reference_column(metadata.to_dataframe(),
+                                reference_column)
+
+    ids_to_keep = get_baseline_donor_md(metadata=metadata,
+                                        reference_column=reference_column,
+                                        time_column=time_column,
+                                        baseline_timepoint=baseline_timepoint)
+
+    filtered_table, = filter(table=table,
+                             metadata=Metadata(ids_to_keep))
+    dataloaf = ancombc(table=filtered_table, metadata=Metadata(ids_to_keep),
```

Comment:
```suggestion
    dataloaf, = ancombc(table=filtered_table, metadata=Metadata(ids_to_keep),
```
If you add a comma here you should be able to get rid of the [0] on the dataloaf below which looks a bit strange. Someone looking at it would probably think "why are they only passing the first one in?"